extensions/Oisthebestletter/sb3ai.js

```
@@ -0,0 +1,311 @@
+// ID: oisthebestletter-sb3ai
+// Description: Use pretrained AI models in your Scratch project!
+// By: Oisthebestletter <https://scratch.mit.edu/users/Oisthebestletter/>
+// License: MPL-2.0
+// Tags: ai llm ml machine clanker bot
+
+function workerMain() {
+  async function importVerifiedModule(url, expectedIntegrity) {
+    const response = await fetch(url, {
+      cache: "no-cache",
+      credentials: "omit",
+    });
```

ESLint

[error] Use Scratch.fetch() instead of fetch()

(extension/use-scratch-fetch)